packages/core/src/components/select/select.story.tsx

```diff
@@ -0,0 +1,54 @@
+import { c, classy, m, SelectProps as BaseProps } from '@onfido/castor';
+import { html } from '../../../../../docs';
+import { Icon } from '../icon/icon.story';
+
+export interface SelectProps extends BaseProps {
+  children?: string | string[] | null;
+  id?: string;
+  placeholder?: string;
+  value?: string;
+}
+
+/**
+ * `.ods-select` uses an `.ods-icon` that requires SVG sprite to be included in
+ * your app.
+ *
+ * https://github.com/onfido/castor-icons
+ */
+export const Select = ({
+  id,
+  placeholder,
+  value,
+  borderless,
+  invalid,
+  children,
+  ...props
+}: SelectProps) =>
+  html('div', {
+    class: classy(c('select'), m({ borderless })),
+    children: [
+      html('select', {
+        ...props,
+        class: classy(
+          c('select-native'),
+          m({
+            borderless,
+            invalid,
+            ...(placeholder && (!value || value === '') && { empty: true }),
+          })
+        ),
+        children: [
+          placeholder &&
+            html('option', {
+              value: '',
+              selected: !value || value === '',
+              disabled: true,
```

Comment:
Should probably be `hidden` instead of `disabled`, arguably.
Behaves more like a Placeholder I suppose.

```suggestion
              hidden: true,
```

Reply:
Why `hidden`?

I want to replicate the React version.

Also, it should still be in the list.

Reply:
I'd assume both versions would see the change.
The arguable part is that the "placeholder option" should not appear in the dropdown list.
If you're sure we want it in the list then `disabled` is good though.

Reply:
OK, I see what you mean.

I personally would be pro showing it, because otherwise when open you don't see a placeholder anymore. However I will consult with a designer.